apps/gateway/src/routes/v2/conversations/chat/chat.handler.test.ts

```diff
@@ -118,7 +118,12 @@ describe('POST /chat', () => {
         messages: [
           {
             role: MessageRole.user,
-            content: 'fake-user-content',
```

Comment:
hmm this should be valid content

Reply:
Not according to this schema
https://github.com/latitude-dev/latitude-llm/blob/main/packages/core/src/constants.ts#L226-L243


This started failing now because before we did this "truquito"

```javascript
import { messageSchema } from '@latitude-data/core/browser'
const messagesSchema = z.array(z.any(messageSchema))
```
☝️ This is using any

What I did was remove the `any`
```javascript
const messagesSchema = z.array(messageSchema)
```

And now types works!

<img width="898" alt="image" src="https://github.com/user-attachments/assets/d4f49a70-a39d-4ecf-8f50-bb022f0218cc" />